src/main/scala/com/cloudant/clouseau/SupportedAnalyzers.scala

```diff
@@ -116,11 +116,11 @@ object SupportedAnalyzers {
       Some(new WhitespaceAnalyzer(IndexService.version))
     case "simple_asciifolding" =>
       Some(new Analyzer() {
-         def createComponents(fieldName: String, reader: Reader): TokenStreamComponents = {
-            val tokenizer: Tokenizer = new LetterTokenizer(IndexService.version, reader);
-            new TokenStreamComponents(tokenizer, new ASCIIFoldingFilter(new LowerCaseFilter(IndexService.version, tokenizer)))
-         }
-       })
+        def createComponents(fieldName: String, reader: Reader): TokenStreamComponents = {
```

Comment:
random white space changes?

Reply:
that's in a separate commit on purpose and is unrelated to snapshots, it was just annoying me as any 'mvn' run also runs the configured code formatter and some previous commit has introduced this formatting error. I can separate it into another PR if you like.